### ISSUE TITLE
[interest invokers] Should interesttarget work on disabled form controls?

### ISSUE THREAD

**Issue by lukewarlow** [openui/open-ui#1002]
@keithamus mentioned this during review of the initial implementation. Disabled buttons right now can't be focused (at least using the tab key with the default `<button disabled>Foo</button>` markup). So interest might only work for specific input modalities if we were to allow disabled elements?

**Comment by lukewarlow** [comment]
@scottaohara I would be especially curious for your input here.

**Comment by scottaohara** [comment]
this seems like it'd legitimize an unfortunate design trend to provide tooltips for disabled controls that often then can't be accessed by everyone since the element is disabled.  

one could argue that the `title` attribute tooltip appears on hover over disabled controls - but... if that person knows anything about the a11y/ux issues of the title attribute, they'd know that's a big opening to rip their argument apart.

**Comment by lukewarlow** [comment]
I'm going to go ahead and close this as something we don't want to allow.

**Comment by lukewarlow** [comment]
Actually will leave this open until the explainer is updated to explicitly disallow disabled buttons.

I think we should also update it to disallow links without a href (as they're effectively disabled).

**Comment by lukewarlow** [comment]
@keithamus did we update the explainer yet? I know I need to update the chrome implementation to disallow anchors without hrefs still

**Comment by keithamus** [comment]
I don't think so. I think we could take stock of a lot of recent decisions and re-work the explainer.
